extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -448,6 +449,25 @@ describe('amp-a4a', () => {
       });
     });
 
+    it('for A4A layout should resolve once FIE is created', () => {
+      a4a.buildCallback();
+      a4a.onLayoutMeasure();
+
+      // Never resolve
+      sandbox.stub/*OK*/(FriendlyIframeEmbed.prototype,'whenIniLoaded')
+          .callsFake(() => {return new Promise(() => {});});
+      let creativeString = buildCreativeString();
+      // TODO: Remove after launch fie_ini_load_fix to 100%
+      creativeString = creativeString.replace('<body>',
+          '<body>' +
+          '<meta name="amp-experiments-opt-in" content="fie_ini_load_fix">');
```

Comment:
If you want to verify query selector is contains instead of exact, you can place text around the 'fie_ini_load_fix' string

Reply:
added